charts/images.yaml

```
@@ -86,7 +86,7 @@ images:
   sourceRepository: github.com/gardener/autoscaler
   repository: eu.gcr.io/gardener-project/gardener/autoscaler/cluster-autoscaler
   tag: "v1.20.1"
-  targetVersion: "<= 1.20"
+  targetVersion: "<= 1.20.x"
```

Thanks for the change! Can we use 

```suggestion
  targetVersion: "< 1.21"
```

to make it idiomatic with the other usages in this file? Looks like we rather prefer to use `<` over `<=` - I was not able to find usages of `<=` in some of the provider extensions.